src/main/kotlin/org/wycliffeassociates/otter/common/data/rxmodel/MimeType.kt

```diff
@@ -0,0 +1,18 @@
+package org.wycliffeassociates.otter.common.data.rxmodel
+
+enum class MimeType(vararg types: String) {
```

Comment:
MimeType is in the java standard library, I had kinda just assumed we'd work off that

Reply:
I think we decided to stick with this for now in today's convo.  Correct?

Reply:
yeah, that's fine